masterlist.yaml

```diff
@@ -7543,6 +7543,7 @@ plugins:
       - name: 'Guard Dialogue Overhaul.esp'
         condition: 'checksum("Guard Dialogue Overhaul.esp", 6ABC1129) or checksum("Guard Dialogue Overhaul.esp", 5CB27D7D) or checksum("Guard Dialogue Overhaul.esp", E248614B) or checksum("Guard Dialogue Overhaul.esp", E40EDE50) or checksum("Guard Dialogue Overhaul.esp", 7D16C40A)'
       - 'Immersive Sounds - Compendium.esp'
+      - 'TAR-GuildMaster1stPersonFix.esp'
```

Comment:
If the changes are included in WACCF, then maybe an `alreadyInOrFixedByX` message would be better?

Reply:
Ah, sorry, I should've gone into more detail as to why I opted for the load after rule over that. The plugin also edits the Thieves Guild leader gloves I assume to make them more consistent with the new model it replaces them with. Doesn't look like the biggest issue if the message is still preferred.

![conflict](https://user-images.githubusercontent.com/69141501/94408206-523dc980-0129-11eb-8da1-328d5a0c4e70.png)